lib/msal-node/README.md

```diff
@@ -81,8 +77,10 @@ npm install @azure/msal-node
 ##  Usage
 
 ### MSAL basics
+- [Understand difference in between Public Client and Confidential Clients](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-client-applications)
 - [Initialize a Public Client Application](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/initialize-public-client-application.md)
-- [Configuration](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md)
+- [Initialize a Confidential Client Application](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/initialize-confidential-client-application.md)
+- [Advanced configuration](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/configuration.md)
```

Comment:
Advanced config should probably go in a separate section than basics

Reply:
Also nitpick: `Advanced Configuration` with capital 'C'

Reply:
Capitalized configuration

Reply:
Also got rid of the "Advanced" part, as that was what it was before.